### PR TITLE
Add main Wiki link to the Overview section in the Welcome intro page

### PR DESCRIPTION
**Pull request by chanseokoh**
Part of #1009.

- Adds a new bundle `c.g.c.t.e.welcome`. (Better name?) This takes most port of the PR.
- Adds a link to our [GitHub wiki page](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/wiki/Google-Cloud-Tools-for-Eclipse) to the *Overview* section. (Look a the `<link>` element in `cloud-tools-for-eclipse.xml`.)

java ee - eclipse platform _005

Looking at the dotted boundary of the `<link>` element, it seems like it already has the top-left area to accommodate an icon:

selection_006

Tried a few ways to put an icon there (using an `` element inside/outside `<link>`, and using a CSS stylesheet to add a background image for `<link>`), but so far, I haven't found a great way to add an icon. 

For the icon file, we might use the existing `gcp-32x32.png`. (BTW, I'm not sure if I have to copy and duplicate the icon file into this bundle.)

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.welcome/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension point="org.eclipse.ui.intro.configExtension">
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.welcome/plugin.xml]: Per http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Fguide%2Fua_intro_minimal.htm
For an icon add another extension point here like this:

   <extension
         point="org.eclipse.ui.intro">
      <intro
            class="org.eclipse.ui.intro.config.CustomizableIntroPart"
            icon="$nl$/icons/image_obj.gif"
            id="org.eclipse.intro.minimal"
            label="Minimal Intro"/>
   </extension>

File: plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: We should add a test that this file is well-formed, like the plugin XML tests in some of the other packages

Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: Test added.

File: plugins/com.google.cloud.tools.eclipse.welcome/plugin.xml (hunk 1 above)
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.welcome/plugin.xml]: I may be wrong, but I think this extension is about the main intro about the product, something you will create when you create a new RCP application. In our IDE case, I think the Eclipse IDE main intro is already there. We are adding a link to this intro (in a top-level page) by extending it, so it seems different.

**Comment by chanseokoh** [comment]
I'm sure the way to put an icon is using CSS. ([Found an example.](https://github.com/ask-fat/eclipse/blob/master/configuration/org.eclipse.osgi/bundles/168/1/.cp/intro/css/overview.css)) I should have known better about CSS styling.

selection_001

I'm afraid we have to copy and duplicate the icon (currently, copied from `.appengine.ui/icons/gcp-32x32.png`) into this bundle in order for CSS to load it.

**Comment by chanseokoh** [comment]
We can also change the wording.

File: plugins/com.google.cloud.tools.eclipse.welcome/intro/css/intro.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+a#cloud-tools-for-eclipse-link img {
+	background-image : url(../icons/gcp-32x32.png);
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.welcome/intro/css/intro.css]: tabs to spaces please, per google style

File: plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+<introContent>
+   <extensionContent path="overview/@" style="css/intro.css">
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: I think you can deindent everything by 1 space

File: plugins/com.google.cloud.tools.eclipse.welcome/plugin.xml (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.welcome/plugin.xml]: 2 space indents please

File: plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<introContent>
+  <extensionContent path="overview/@" style="css/intro.css">
+    <group style-id="content-group">
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: When installed, the Welcome will be opened on the _What's New_ page (`path="whatsnew/@"`), and the new contributions are subtly highlighted on the page.

There's sometimes a Migration page offered too (`path="migrate/@"`) and we could link to the _Migrating from GPE_ page.

Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml]: Okay, let's add these pages on subsequent PRs.

**Pull request merged**